### ISSUE TITLE
Failing basic setup

### ISSUE THREAD

**Issue by zettam** [laktek/punch#125]
npm install punch -g
punch setup mysite
cd mysite
punch s

--getting the following error:

Failed to read the configurations.
If you have already created a config file, please check it for any syntax errors.

**Comment by JimmiLee** [comment]
It's because punch installs the files in a default directory, so before the command `punch s` you need to cd into your directory with cd mysite/default like so:

```
npm install punch -g
punch setup mysite
cd mysite
cd default
punch s
```

**Comment by bretonio** [comment]
Yes it does create a default folder where `punch s` starts up the tutorial on the server.

(Nice work on the tutorial by the way. Look ma I am not just surging, I am learning, too!)

 Perhaps a simple edit to the docs to reflect the fact?
